src/base.config.ts

```diff
@@ -193,6 +194,24 @@ export default function webpackConfigFactory(args: any): WebpackConfiguration {
 		}
 	};
 
+	const postcssImportConfig = {
+		filter: (path: string) => {
+			return /.*variables(\.m)?\.css$/.test(path);
+		},
+		resolve: (id: string, basedir: string, importOptions: any = {}) => {
```

Comment:
A custom resolver so that users can still use `~` to import `variables.css` from node_modules, `postcss-import` does not support this prefix and performs it's own resolution but we need to be able to consistently support imports